tests/integration/helpers/API-service/enum/mock-scenario.enum.ts

```diff
@@ -24,10 +24,11 @@ export enum DroughtScenario {
 }
 
 export enum TyphoonScenario {
-  NoEvent = 'noEvent',
-  EventNoLandfall = 'eventNoLandfall',
-  EventNoLandfallYet = 'eventNoLandfallYet',
-  EventNoTrigger = 'eventNoTrigger',
-  EventTrigger = 'eventTrigger',
-  EventAfterLandfall = 'eventAfterLandfall',
+  Trigger = 'trigger',
+  NoTrigger = 'no-trigger',
+  Warning = 'warning',
+  NoLandfallTrigger = 'no-landfall-trigger',
+  NoLandfallYetWarning = 'no-landfall-yet-warning',
+  OngoingTrigger = 'ongoing-trigger',
+  WarningAndTrigger = 'warning-and-trigger',
```

Comment:
Instead of a separate scenario for `warning-and-trigger`, is it possible to use `removeEvents: false` and call the mock endpoint once with `scenario: trigger` and again with `scenario: warning`?

Reply:
in other disaster-types we also have a lot of multi-event scenarios, even as the default/trigger scenario (in both floods/UGA and flash-floods/MWI). Changing the setup like you propose would also prevent to easily mock those trigger scenarios from the front-end.

So for now I propose to leave it, but we can re-evaluate at any moment.